### ISSUE TITLE
[BUG] [CRASH] [Bromite] Uncaught Reference: Atomics is not defined

### ISSUE THREAD

**Issue by Danoloan10** [Etwas-Builders/modfy.video#87]
I got this error while browsing in Bromite version 85.0.4183.114 in Android 9 (Lineage OS + microG)

-----
 Browser Version: Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/85.0.4183.114 Safari/537.36 

-----
 **Error Message : Uncaught ReferenceError: Atomics is not defined** 

-----
 Call Stack : undefined

**Comment by CryogenicPlanet** [comment]
Thanks for opening this issue, but we don't fully support mobile browsers yet. To support some of the underlying technologies need browser flags to be turned on manually.

![](https://caniuse.bitsofco.de/image/sharedarraybuffer.jpg)

**One question tho did you manually go to app.modfy.video or by chance was this a tablet?** because on mobile devices modfy.video(the landing page) shouldn't give you access to the application

**Comment by Danoloan10** [comment]
I manually went there yes, by ticking the "desktop view" in the browser; I opened this issue as it was an error that I did not find anywhere, so that it was registered somewhere.

**Comment by CryogenicPlanet** [comment]
@Danoloan10 Yeah that makes sense, I will try to explore how to fix the Atomics error but wouldn't except a resolution anytime soon
